distribution/scenes/templates/Textures_Materials/Metals/__index.htm

```diff
@@ -359,9 +359,8 @@
 <hr size="3">
 <table width="95%" align="center">
 <tr>
- <td><i><font size="-1">&copy; Friedrich A. Lohm&uuml;ller, 2010<br>
-
-   homepage:<a href="http://www.f-lohmueller.de"><nobr>http://www.f-lohmueller.de</nobr></font></i></a>
+ <td><i><font size="1">&copy; Persistence of Vision Ray Tracer, 2017<br>
```

Comment:
You have a point there that I didn't notice, and I suspect you're right. But when it comes to legal stuff, I think we should be cautious, and at least check with Chris, if not with Friedrich.